scripts/ci/in_container/kubernetes/docker/airflow-test-env-init.sh

```diff
@@ -20,6 +20,7 @@ set -x
 
 cd /opt/airflow/airflow && \
 cp -R example_dags/* /root/airflow/dags/ && \
+find /root/airflow/dags/ && \
```

Comment:
This is just to show what dags are there for debugging right? :+1:

Reply:
Yel